tools/devops/automation/post-build-pipeline.yml

```diff
@@ -0,0 +1,53 @@
+# YAML pipeline for post build operations. 
+# This pipeline will trigger automatically after successful completion of the `build_packages` stage against the specified branches.
+
+trigger: none
+pr: none
+
+resources:
+  pipelines:
+  - pipeline: macios
+    source: xamarin-macios
+    trigger:
+      branches:
+      - main
+      - 6.0.*
+      stages:
+      - build_packages
+
+jobs:
+- job: post_build
+  displayName: Post Build
+  pool:
+    vmImage: windows-2019
+  workspace:
+    clean: all
+  variables:
+  - group: Publish-Build-Assets
+  steps:
+  - checkout: self
+    clean: true
+
+  - task: UseDotNet@2
+    inputs:
+      version: 5.0.200
+
+  - download: macios
+    artifact: AssetManifests
+
+  - script: make -C $(Build.SourcesDirectory)/dotnet targets/Microsoft.iOS.Sdk.Versions.props
+    displayName: make Microsoft.iOS.Sdk.Versions.props
+
+  - powershell: >-
+      & dotnet build -v:n -t:PushBARManifest
+      -p:BarManifestOutputPath=$(Agent.BuildDirectory)\macios\AssetManifests -p:BuildAssetRegistryToken=$(MaestroAccessToken)
+      $(Build.SourcesDirectory)\dotnet\package\Microsoft.iOS.Ref\package.csproj
+    displayName: generate and publish BAR manifest
```

Comment:
I don't know this syntax. Why not powershell: | ?

Reply:
The `>` replaces new lines with spaces, while `|` preserves new lines. Also the `-` is probably not strictly needed, it just ensures that there is no trailing newline.  I like to use `>-` as a way to separate out a long list of build arguments over multiple lines rather than having to put them all on one line.

Reply:
Did not know that trick, nice!